unix/w0vncserver/wlr/WlrPixelBuffer.cxx

```diff
@@ -111,3 +116,19 @@ void WlrPixelBuffer::captureFrameDone()
 
   server->add_changed({{0, 0, width(), height()}});
 }
+
+void WlrPixelBuffer::resize()
+{
+  WlrScreencopyManager::resize();
+
+  setBuffer(output->getWidth(), output->getHeight(), getBufferData(),
+            output->getWidth());
+
+  server->setPixelBuffer(this);
+
+  if (captureInProgress)
+    server->unblockUpdates();
+  captureInProgress = false;
+
+  captureFrame();
```

Comment:
This looks suspicious. If a capture is in progress, then I assume it needs to be aborted? And the buffer is not valid at this point?